docs/_static/style.css

```diff
@@ -102,3 +102,64 @@ code > .indName {
 #incVirtModLbl {
     display: inline;
 }
+
+/* extend furo for inline ".only-dark" elements */
```

Comment:
Are there any elements below here that we could move to the `conf.py`? Maybe the directives that style things handled by sphinx extensions/plugins?

Reply:
everything here is standard css, and will work out of the box with any package. except the last bit, 
```
button.copybtn.copybtn svg {
  stroke: var(--color-content-foreground);
}
```
but it will be black if --color-content-foreground is not defined.

Reply:
There's a `--color-foreground-primary` (and `secondary`) found in the `furo.css`. We can use that, I'm sure.

Reply:
yes, but you're worried about maintenance issues. We can use the variables we self-define in conf.py, but we *do* need to use a var somewhere (unless we replace classes with js... which is just unnecessary here)

Reply:
Okay, so long as we're building off of the `conf.py` configuration, I'm happy. Thanks!